lib/msf/core/option_container.rb

```diff
@@ -725,6 +725,12 @@ def validate(datastore)
         "One or more options failed to validate", caller
     end
 
+    # message about missing SSL
+    if datastore.include?('RPORT') && [443, 8443].include?(datastore['RPORT']) &&
+      datastore.include?('SSL') && datastore['SSL'] == false
+      puts('If you are testing a SSL enabled port be sure to execute \'set SSL true\'')
```

Comment:
Yeah I felt the same, but I see no other way in warning the user. If I raise an exception, the rest of the validation will fail. So I can only put it at the end of the validate function and set the validate result to true in the rescue block (because validate will not return true). But this is more a code style break :(